Tests/FeatherWidgets.TestIntegration/Navigation/LanguageSelectorWidgetTests.cs

```diff
@@ -157,6 +329,12 @@ private string GetPageUrl(string pageName, CultureInfo culture, bool isDefaultCu
             }
         }
 
+        private string GetPageUrlOfNotTranslatedPages(string homePageName, CultureInfo culture, CultureInfo homePageCulture)
+        {
+            // returns /tr-tr/TestPagetr-TR
```

Comment:
The comment is not correct. It is just copy/paste from the other method. The comments are very important because they have to help others to understand the code not to confuse you.